x-pack/solutions/security/packages/navigation/src/i18n_strings.ts

```diff
@@ -154,7 +150,7 @@ export const i18nStrings = {
     defaultMessage: 'Entity Store',
   }),
   entityAnalytics: {
-    landing: i18n.translate('securitySolutionPackages.navLinks.entityAnalytics', {
+    title: i18n.translate('securitySolutionPackages.navLinks.entityAnalytics', {
       defaultMessage: 'Entity Analytics',
     }),
```

Comment:
I would consider fixing the text casings in this PR. See https://docs.google.com/spreadsheets/d/1UFzNw-BBfOnY0GyFz5sC57uVi7bSQU1dYeuAjszWUww/edit?pli=1&gid=0#gid=0 for a (partial) guide.

Reply:
I already reached out to the Entity Analytics team about this, they are fixing it [here](https://github.com/elastic/kibana/pull/217350)